app/controllers/reset_passwords_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ResetPasswordsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
Dynamic finders are not recommended anymore. You should use `find_by`, since it does not use `method_missing`. It is faster, because of that.

```ruby
user = User.find_by(email: params[:email])
```